.github/workflows/ci-test.yml

```diff
@@ -79,6 +79,56 @@ jobs:
       - name: build image with dev-setup.sh
         run: docker build -f docker/ci/${{ matrix.target_os }}/Dockerfile -t diem/build_environment:test .
 
+  build-images:
+    runs-on: ubuntu-latest-xl
+    continue-on-error: false
+    needs: prepare
+    if: ${{ github.event_name == 'push' && needs.prepare.outputs.test-rust == 'true' }}
+    env:
+      TAG: github-1
+      DOCKERHUB_ORG: libra
```

Comment:
`diem`?

Reply:
I didn't see this env var being used in the job def.

Reply:
oh, you are right